python/cm/migrations/0068_alter_tasklog_action.py

```diff
@@ -15,6 +15,15 @@
 from django.db import migrations, models
 import django.db.models.deletion
 
+def fix_joblog(apps, schema_editor):
+    JobLog = apps.get_model('cm', 'JobLog')
+    for job in JobLog.objects.all():
```

Comment:
should be single query, something like `JobLog.objects.filter(task=None).delete()`

Reply:
It is done in last commit before your review